config/mappings/test_mapping_config.py

```diff
@@ -0,0 +1,13 @@
+import json
+from jsonschema import validate
+from mapping_schema import top_level_assessment_mapping_schema as schema
+
+with open("config/mappings/mapping.json", "r") as mapping_file:
+    mapping = json.load(mapping_file)
+
+try:
+    validate(instance=mapping, schema=schema)
+except Exception:
+    raise Exception
+finally:
+    print("Mapping conforms with schema.")
```

Comment:
Doesn't finally run even if validation fails? So I think you need to move the `print("Mapping conforms with schema.")` inside the `try:` clause